src/server/game/Entities/GameObject/GameObject.h

```diff
@@ -772,6 +775,7 @@ class TC_GAME_API GameObject : public WorldObject, public GridObject<GameObject>
         uint32 m_lootRecipientGroup;
         uint16 m_LootMode;                                  // bitmask, default LOOT_MODE_DEFAULT, determines what loot will be lootable
         uint32 m_lootGenerationTime;
+        ObjectGuid m_linkedTrap;
```

Comment:
Okay but I'd want the system to actually be implemented then rather than just have functions with no effect.

Reply:
Okay my bad it was actually done here.